toolkit/imageconfigs/full.json

```diff
@@ -17,8 +21,12 @@
             "Name": "CBL-Mariner Core",
             "PackageLists": [
                 "packagelists/hyperv-packages.json",
-                "packagelists/core-packages-image.json"
+                "packagelists/core-packages-image.json",
+                "packagelists/selinux.json"
```

Comment:
Let's not include selinux.json or the SELinux KernelCommandLine for the "CBL-Mariner Core" option to keep it in line with the changes from #2313.